src/components/firebase-access.jsx

```diff
@@ -226,26 +268,45 @@ function randomDate(start, end) {
     return date;
 }
 
-export const setUpRoom = (numOfSymbols,rounds,password,startingMoney = 10000) => {
-
-    const roomRef = db.collection('Rooms').doc();
-    roomRef.set({
-        day_index: 0,
-        phase: 'no-host',
-        password: password,
-        starting_money: startingMoney,
-    });
-    const roomID = roomRef.id;
-    initSymbols(null,null,numOfSymbols).then((symbolsL) => {
-        initDates(symbolsL,rounds).then((datesD)=> {
-            roomRef.update({
-                symbols: symbolsL,
-                dates: datesD["dates"],
-            });
-            initializeQuiz(symbolsL,roomID,datesD["period"],datesD["dates"]);
-        });
-    });
-    return roomID;
+// export const setUpRoom = (numOfSymbols,rounds,password,startingMoney = 10000) => {
+//
+//     const roomRef = db.collection('Rooms').doc();
+//     roomRef.set({
+//         day_index: 0,
+//         phase: 'no-host',
+//         password: password,
+//         starting_money: startingMoney,
+//     });
+//     const roomID = roomRef.id;
+//     initSymbols(db,null,null,null,numOfSymbols).then((symbolsL) => {
+//         initDates(symbolsL,rounds).then((datesD)=> {
+//             roomRef.update({
+//                 symbols: symbolsL,
+//                 dates: datesD["dates"],
+//             });
+//             initializeQuiz(symbolsL,roomID,datesD["period"],datesD["dates"]);
+//         });
+//     });
+//     return roomID;
+// }
+
+export const setUpRoom = (symbolsL,Rounds,password,startingMoney = 10000) => {
+  const roomRef = db.collection('Rooms').doc();
+  roomRef.set({
+      day_index: 0,
+      phase: 'no-host',
+      password: password,
+      starting_money: startingMoney,
+  });
+  const roomID = roomRef.id;
+  initDates(symbolsL,Rounds).then((datesD)=> {
```

Comment:
space before =>

Reply:
also please use better variable names than 'symbolsL', it's unclear what 'L' stands for